doc/man3/OSSL_ENCODER.pod

```diff
@@ -114,6 +108,12 @@ otherwise 0.
 
 OSSL_ENCODER_number() returns an integer.
 
+=head1 NOTES
+
+OSSL_ENCODER_fetch() may be called implicitly by other fetching
+functions, using the same library context and properties.
```

Comment:
I know it's an existing bit of text but I don't like _the same_ here.  Perhaps _their_ ?